app/main.py

```diff
@@ -1,6 +1,21 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
```

Comment:
Following the checklist, you need to use uppercase for constants' names. However, 'CITY' is not a constant because it could vary depending on the city you want to get the weather for. It would be better to pass the city name as a parameter to the 'get_weather' function.